### DISCUSSION TITLE
Nonexpanding discord server

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#226] [General]
I'm not trying to step on anyone's toes, or take anything away from ACG by posting this. I like this place and the interesting characters here.

I'm also pretty new to discord, as it was recently recommended to me by a viewer of my YouTube channel. It seems to me something like a glorified IRC server. If that interests anyone, here is the invite for the "Nonexpanding" server:

https://discord.gg/kAcYcRwG

**Comment by RedshiftDrift** [comment]
@mikehelland, great initiative!  *discord* was proposed last year as a new platform for ACG.  However, since ACG also maintains a [website](https://cosmology.info/), GitHub was chosen instead to host both the website and discussions.

If you see something interesting on *Nonexpanding* that ACG doesn't have, don't forget to send it over here! 😄

**Comment by LifeIsStrange** [comment]
@mikehelland the invite link has expired it seems

**Reply by mikehelland** [reply]
Whoops, I thought I clicked on the never expire:

https://discord.gg/rXrGbzAW

Note, there are just a couple folks there, and not much activity. Maybe that will change. But this group (ACG) is much more fruitful.

Cheers!

**Reply by LifeIsStrange** [reply]
Thanks
